addon/components/infinity-loader.js

```diff
@@ -14,8 +14,21 @@ export default Ember.Component.extend({
   developmentMode: false,
   scrollable: null,
 
+  didRender() {
+    this._super(...arguments);
+    if(Ember.VERSION >= '1.13.0') {
+      this._setup();
+    }
+  },
+
   didInsertElement() {
     this._super(...arguments);
+    if(Ember.VERSION < '1.13.0') {
```

Comment:
Hey!

@mike-north: Thanks for the heads-up, I'll change this asap.

 @hhff: what would be your preferred way of version checking, using the regex or your package?

Cheers